assessment-api/open-api-specification.yml

```diff
@@ -40,14 +103,19 @@ components:
       properties:
         id:
           type: 'string'
-        owner_id:
+        provider_id:
           type: 'string'
-        target_id:
+        case_reference_number:
           type: 'string'
-        assessment:
+        name:
```

Comment:
Is name actually type?

Reply:
well the object is called assessmentDetail 

'name' is the name of the assessment

Reply:
it also follows a similar structure to the other objects,  attributs have a name, entity have a name, makes sense that assessment does.